maestro-orchestra/src/test/java/maestro/orchestra/yaml/YamlCommandReaderTest.kt

```diff
@@ -82,29 +83,6 @@ internal class YamlCommandReaderTest {
         assertThat(e.message).contains("Flow files must contain a config section and a commands section")
     }
 
-    @Test
-    fun initFlow(
-        @YamlFile("007_initFlow.yaml") commands: List<Command>,
```

Comment:
lets not completely remove those tests , rather test that they throw an exception, you can copy from my PR

Reply:
Done ✅